swift/apple/FirezoneKit/Sources/FirezoneKit/Views/SessionView.swift

```diff
@@ -130,22 +130,35 @@ struct ResourceSection: View {
   var body: some View {
     ForEach(resources) { resource in
       HStack {
-        NavigationLink { ResourceView(model: model, resource: resource) }
-      label: {
-        HStack {
-          Text(resource.name)
-          if resource.canBeDisabled {
-            Spacer()
-            Toggle("Enabled", isOn: Binding<Bool>(
-              get: { model.isResourceEnabled(resource.id) },
-              set: { newValue in
-                model.store.toggleResourceDisabled(resource: resource.id, enabled: newValue)
-              }
-            )).labelsHidden()
+        if !resource.isInternetResource() {
+            NavigationLink { ResourceView(model: model, resource: resource) }
+          label: {
+            ResourceLable(resource: resource, model: model )
           }
+        } else {
+          ResourceLable(resource: resource, model: model)
         }
       }
       .navigationTitle("All Resources")
+    }
+  }
+}
+
+struct ResourceLable: View {
+  let resource: Resource
+  @ObservedObject var model: SessionViewModel
+
+  var body: some View {
+    HStack {
+      Text(resource.name)
+      if resource.canBeDisabled {
+        Spacer()
+        Toggle("Enabled", isOn: Binding<Bool>(
```

Comment:
Does this `Enabled` text show in the UI? I was going to suggest removing it since it's redundant with the visual state of the switch.

Reply:
the `labelIsHidden` below hides it from the UI.